packages/functions/package.json

```diff
@@ -2,9 +2,11 @@
   "name": "@vercel/functions",
   "description": "Runtime functions to be used with your Vercel Functions",
   "homepage": "https://vercel.com",
-  "version": "1.0.2",
-  "types": "index.d.ts",
-  "main": "index.js",
+  "version": "1.1.0",
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏revert version change

```suggestion
  "version": "1.0.2",
```

We use changesets to manage the version. Can we revert this change in this PR?